### ISSUE TITLE
Add Views Grid Three and Views Grid Two classes

### ISSUE THREAD

**Issue by aaroncole** [SU-SWS/open_framework#283]
The .views-grid-n layout options are awesome.  Having the ability to create two and three column grids in the same manner would be even awesomer!  Is there any reason not to?

**Comment by meganem** [comment]
You are correct! We should add those in. Just takes writing the code for those.
You want to share yours? Feel free to paste into the comments here, or do a more formal pull request if you like.
